<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>OptStatus</class>
 <widget class="QWidget" name="OptStatus" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OptStatusUI</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>Prompt for status message when choosing</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="ck_askOnline" >
        <property name="text" >
         <string>Online</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ck_askOffline" >
        <property name="text" >
         <string>Offline</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="2" column="1" >
      <widget class="QSpinBox" name="sb_asOffline" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QLabel" name="lb_asOffline" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Minutes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLabel" name="lb_asXa" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Minutes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QSpinBox" name="sb_asXa" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="lb_asAway" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Minutes</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QCheckBox" name="ck_asOffline" >
       <property name="text" >
        <string>Auto offline after:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QCheckBox" name="ck_asXa" >
       <property name="text" >
        <string>Auto XA after:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QCheckBox" name="ck_asAway" >
       <property name="text" >
        <string>Auto away after:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QSpinBox" name="sb_asAway" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="TextLabel1_8" >
     <property name="text" >
      <string>Auto status message:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="te_asMessage" >
     <property name="maximumSize" >
      <size>
       <width>32767</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox10" >
     <property name="title" >
      <string>Presets</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Preset:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_preset" />
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pb_spNew" >
          <property name="text" >
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_spDelete" >
          <property name="text" >
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_sp_status" />
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>97</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Priority:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_sp_priority" >
          <property name="maximumSize" >
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="TextLabel2_2" >
        <property name="text" >
         <string>Content:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="te_sp" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>32767</width>
          <height>90</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>ck_asAway</tabstop>
  <tabstop>sb_asAway</tabstop>
  <tabstop>ck_asXa</tabstop>
  <tabstop>sb_asXa</tabstop>
  <tabstop>ck_asOffline</tabstop>
  <tabstop>sb_asOffline</tabstop>
  <tabstop>te_asMessage</tabstop>
  <tabstop>pb_spNew</tabstop>
  <tabstop>pb_spDelete</tabstop>
  <tabstop>te_sp</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ck_asOffline</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_asOffline</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>65</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel" >
     <x>171</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ck_asXa</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_asXa</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>95</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel" >
     <x>172</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ck_asAway</sender>
   <signal>toggled(bool)</signal>
   <receiver>sb_asAway</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>61</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel" >
     <x>180</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
